helm/managed/logging/loki-stack/README.md

```diff
@@ -182,8 +185,10 @@ Example use:
 
 The grafana hpcc logaccess values should provide Grafana connection information, such as the host, and port; the Loki datasource where the logs reside; the k8s namespace under which the logs were created (non-default namespace highly recommended); and the hpcc component log format (table|json|xml)
 
+Example values file describing logAccess targeting loki stack deployed as 'myloki4hpcclogs' on the default namespace. The host entry needs to reflect the actual host of the target Loki stack:
```

Comment:
I added a bit to the ending , trying to connect the significance of the deployment name to the hostname:

Example values file describing logAccess targeting loki stack deployed as 'myloki4hpcclogs' on the default namespace. Note that the "host" entry must reflect the name of the deployed Loki stack, as shown in the excerpt below (eg **_myloki4hpcclogs_**-grafana.default.svc.cluster.local):